### PR TITLE
Enchancement/add to blog post

### LINKED ISSUE #23: Typo in website description
In main.swift

### PR DESCRIPTION
**Pull request by BennyDeBock**
- Added dates to the list of blog posts
- Added date and author to individual blogposts
- Fixed a typo in the sites description

closes #23 #17

### CODE REVIEW

File: Content/posts/updating-template-to-async-await.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ date: 2022-02-22 10:22
 description: We're updating Vapor's template to use async/await
 tags: template, async/await
 author: Tim
+leggings: this shit be ballin
```
Comment (0xTim) [inline Content/posts/updating-template-to-async-await.md:6]: 😂 I'm guessing this is probably left over from testing

File: Sources/Blog/VaporTheme.swift [hunk 2]
```diff
@@ -9,6 +10,12 @@ extension Theme where Site == Blog {
 
 private struct VaporThemeHTMLFactory: HTMLFactory {
     typealias Site = Blog
+    var dateFormatter: DateFormatter
+    init() {
+         dateFormatter = DateFormatter()
```
Comment (0xTim) [inline Sources/Blog/VaporTheme.swift:15]: Nit spacing
```suggestion
        dateFormatter = DateFormatter()
```

File: Sources/Blog/VaporTheme.swift [hunk 3]
```diff
@@ -9,6 +10,12 @@ extension Theme where Site == Blog {
...
+
```
Comment (0xTim) [inline Sources/Blog/VaporTheme.swift:16]: Remove this line

File: Sources/Blog/main.swift [hunk 4]
```diff
@@ -2,6 +2,9 @@ import Foundation
 import Publish
 import Plot
 
+// TODO: Add date to individual post
```
Comment (0xTim) [inline Sources/Blog/main.swift:5]: You can remove these

File: Sources/Blog/main.swift [hunk 5]
```diff
@@ -11,12 +14,14 @@ struct Blog: Website {
 
     struct ItemMetadata: WebsiteItemMetadata {
         // Add any site-specific metadata that you want to use here.
+        var author: String
+        var leggings: String?
```
Comment (0xTim) [inline Sources/Blog/main.swift:18]: Again left over

**Review by 0xTim — CHANGES_REQUESTED** [review]
Few tweaks but looks great overall!

File: Sources/Blog/main.swift (hunk 5 above)
Comment (BennyDeBock) [reply Sources/Blog/main.swift:18]: the leggings is definitely left over, but the author i'm using in the vaportheme.swift to display it

**Review by 0xTim — APPROVED** [review]
LGTM

**Pull request merged**